modules/assim.sequential/man/NA_preprocess.Rd

```diff
@@ -0,0 +1,30 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
seems like this file should have gone away when the function name was changed. If it was deleted in your branch make sure to git rm it from the PR too.

Reply:
Addressed in [d751ffc](https://github.com/PecanProject/pecan/pull/3321/commits/d751ffc60fad5c5dbf9030c0e9b938a65096dc36) commit